app.js

```diff
@@ -65,44 +68,77 @@ adminAppRouter.get('/', async (req, res) => {
 });
 
 /**
- * @summary Get APIs
+ * @summary Get account indexes
  */
-appRouter.get(`/${api}`, async (req, res) => {
+appRouter.get(`/${accountIndexEndpoint}`, async (req, res) => {
   try {
-    const MAX_PAGE_SIZE = 500;
-    const { page } = req.query;
-    /**
-     * Return 400 errors if page[size]/page[number] are not valid
-     */
-    if (page) {
-      const { size, number } = page;
-      const isInvalidSize = (size !== '') && (size <= 0 || size > MAX_PAGE_SIZE);
-      const isInvalidNumber = (number !== '') && number <= 0;
-      const errors = [];
-
-      if (isInvalidSize || isInvalidNumber) {
-        if (isInvalidSize) errors.push(`page[size] should an integer between 1 to ${MAX_PAGE_SIZE}.`);
-        if (isInvalidNumber) errors.push('page[number] should an integer starts at 1.');
-        return res.status(400).send(badRequest(errors));
+    const { accountIndexCode, organizationCode } = req.query;
+    if (!accountIndexCode && !organizationCode) {
+      res.status(400).send(badRequest(['At least one parameter is required.']));
+    } else {
+      // If there's additional parameters in the query object, oracle will complain
+      // because it doesn't recognize them. Thus, we only add the ones we need,
+      // and the ones that are defined.
+      // TODO: There's got to be a less ugly way to do this.
+      const params = {};
+      if (accountIndexCode) {
+        params.accountIndexCode = accountIndexCode;
+      }
+      if (organizationCode) {
+        params.organizationCode = organizationCode;
       }
+      const result = await db.getAccountIndexes(params);
+      res.send(result);
     }
+  } catch (err) {
+    errorHandler(res, err);
+  }
+});
 
-    const result = await db.getApis(req.query);
-    return res.send(result);
+/**
+ * @summary Get API by unique ID
```

Comment:
This description should be changed.

Reply:
Ah, good catch. There's a lot of things that need to be changed from the skeleton.